.github/release_template.md

```diff
@@ -0,0 +1,33 @@
+<!--
+This is the template to release a new version on the App Store
+-->
+
+Release a new version
+## TASKS:
+- [ ] [GIT] Create branch release_X.X.X (freeze the code)
+- [ ] [DEV] Update version number X.X.X
+- [ ] [DIS] Update changelogs:
+     - [ ] [DEV] Update changelog https://github.com/owncloud/ios-app/blob/master/CHANGELOG.md
+     - [ ] [GIT] Add the new release on GitHub https://github.com/owncloud/ios-app/releases
+- [ ] [QA] Design Test plan
+- [ ] [QA] Regression Test plan
+- [ ] [DIS] Update screenshots if needed
+- [ ] [DIS] Release to appstore
```

Comment:
Probably missing before that step:
- the submit-to-App-Store step
- documentation on how an approved version should be released (immediately, by developer, staged rollout, ..), who makes that decision, or how that decision is made